server/helios/migrations/0002_qrcode.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
remove all migrations and regenerate them all as one

`rm <path to migration>`
`python manage.py makemigrations`

Did you already run `manage.py migrate`? If so, check database for already applied migrations

Reply:
Why? Regenerating migrations is not generally a good idea.

Reply:
I did it once, because the old helios migrations were handmade and didn't work after migrating to 1.11

Reply:
I meant all migrations that correspond to @piotrkawa changes. Piotr told me he generated bunch of them for those changes.

Reply:
Done